src/components/select/select.js

```diff
@@ -228,9 +228,10 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
         });
       }
 
-      if (formCtrl) {
+      if (formCtrl && angular.isDefined(attr.multiple)) {
```

Comment:
Yes that is valid, as you can see in 09bd5a3b52f29dfe530192099d2c405f204ec3ad, we should only set the form to pristine if `multiple` objects are preselected in the `ngModel`.
I manually tested that too, if there is a predefined value for a `single select`, we don't need to set the form pristine. It only sets the form to dirty if we render `multiple`